src/server/local/index.test.js

```diff
@@ -24,9 +24,6 @@ describe('Local Server', () => {
     ${'Migrated Home Page'}            | ${'/pidgin.json'}                                                | ${'/pidgin/homePage/index.json'}
     ${'Home Page'}                     | ${'/kyrgyz/tipohome.json'}                                       | ${'/kyrgyz/homePage/index.json'}
     ${'Most Read'}                     | ${'/pidgin/mostread.json'}                                       | ${'/pidgin/mostRead/index.json'}
-    ${'Most Read with variant'}        | ${'/zhongwen/mostread/trad.json'}                                | ${'/zhongwen/mostRead/trad.json'}
```

Comment:
Did we mean to delete this? 

```suggestion
| ${'Most Read with variant'} | ${'/zhongwen/mostread/trad.json'} | ${'/zhongwen/mostRead/trad.json'}
```

Reply:
oops no. Thanks.

Reply:
put back